frontend/src/components/SyncEditor/SyncEditor.css

```diff
@@ -0,0 +1,88 @@
+:root {
+    --sy-c-code-editor__code--PaddingTop: 0;
+    --sy-c-code-editor__code--PaddingRight: 0;
+    --sy-c-code-editor__code--PaddingBottom: 0;
+    --sy-c-code-editor__code--PaddingLeft: 0;
+}
+
+.sy-c-code-editor__title {
+    display: flex;
+    align-items: center;
+    padding: 0 12px;
+}
+
+.pf-c-code-editor__container {
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+}
+
+.pf-c-code-editor__container .pf-c-code-editor__header {
+    font-size: 14px;
+    font-weight: 600;
+    color: #ededed;
+    background-color: #25282c;
+    border-bottom: 1px solid #55585c;
+}
+
+.pf-c-code-editor__container .pf-c-code-editor__controls {
+    width: 100%;
+    justify-content: space-between;
+}
+
+.pf-c-code-editor__container .pf-c-code-editor__header::before {
+    border: none;
+}
+
+.pf-c-code-editor__container .pf-c-code-editor__controls .pf-c-button.pf-m-control {
+    background-color: #25282c;
+}
+
+.pf-c-code-editor__container .pf-c-button.pf-m-plain,
+.pf-c-code-editor__controls .pf-c-button.pf-m-control:not([disabled]) {
+    color: #ededed;
+}
+
+.pf-c-code-editor__container .pf-c-button::after {
+    border: none;
+    border-left: 1px solid #55585c;
+}
+
+.pf-c-code-editor__container .pf-c-button:hover {
+    color: #ededed;
+    background-color: #25282c;
+}
+
+.pf-c-code-editor__container .pf-c-code-editor__main {
+    border: none;
+}
+
+.pf-c-code-editor__container .pf-c-code-editor__code {
+    padding: var(--sy-c-code-editor__code--PaddingTop) var(--sy-c-code-editor__code--PaddingRight)
+        var(--sy-c-code-editor__code--PaddingBottom) var(--sy-c-code-editor__code--PaddingLeft);
+}
```

Comment:
These seem like they will affect PF styles globally. Can we scope these and use variables where PF provides them?

Reply:
pf-c-code-editor__container was actually my class to contain the PF CodeEditor--I'm changing to sync-editor__container to be less confusing